sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -814,14 +937,31 @@ def _plot_ice_lines(
             n_ice_to_plot,
             replace=False,
         )
+
+        individual_line_kw_copied = (
+            individual_line_kw.copy()
+        )  # copy, so that color list can be deleted and won't throw
+        # an error when passed to ax.plot()
+        color_list = individual_line_kw.get("color_list", None)
+        if color_list:
+            if len(color_list) != preds.shape[0]:
```

Comment:
wenn du unten das `del` weg machst, können die zwei if statements mit `and` verbunden werden